daemon/src/db.rs

```diff
@@ -66,98 +71,39 @@ pub async fn load_order_by_id(
     id: OrderId,
     conn: &mut PoolConnection<Sqlite>,
 ) -> anyhow::Result<Order> {
-    let uuid = serde_json::to_string(&id).unwrap();
-
-    let row = sqlx::query!(
-        r#"
-        select * from orders where uuid = ?;
-        "#,
-        uuid
-    )
-    .fetch_one(conn)
-    .await?;
-
-    let uuid = serde_json::from_str(row.uuid.as_str()).unwrap();
-    let trading_pair = serde_json::from_str(row.trading_pair.as_str()).unwrap();
-    let position = serde_json::from_str(row.position.as_str()).unwrap();
-    let initial_price = serde_json::from_str(row.initial_price.as_str()).unwrap();
-    let min_quantity = serde_json::from_str(row.min_quantity.as_str()).unwrap();
-    let max_quantity = serde_json::from_str(row.max_quantity.as_str()).unwrap();
-    let leverage = Leverage(row.leverage.try_into().unwrap());
-    let liquidation_price = serde_json::from_str(row.liquidation_price.as_str()).unwrap();
-    let creation_timestamp = serde_json::from_str(row.creation_timestamp.as_str()).unwrap();
-    let term = serde_json::from_str(row.term.as_str()).unwrap();
-    let origin = serde_json::from_str(row.origin.as_str()).unwrap();
-    let oracle_event_id = OracleEventId(row.oracle_event_id);
+    let row = sqlx::query_file!("src/db_sql/query_load_order_by_id.sql", id)
```

Comment:
I'd prefer if all the SQL files are next to the rust files and not in their own directory.